AnnoyancesFilter/Popups/sections/subscriptions_specific.txt

```diff
@@ -119,6 +119,8 @@ estadao.com.br#?#.content-wrapper > div[class^="styles__Container-"]:has(> div[c
 putlockert.com###cookiedata
 earthly.dev#?#div[class] > section:has(> div.px-6 > div.block > div.newsletter)
 ||static.hokkaido-np.co.jp/*/bnr_news_letter_small.png
+||benesse.ne.jp/*/img/pc/banner/line_account_2111_600x514
```

Comment:
The rule fails on mobile. `||benesse.ne.jp/*/banner/line_account_2111_600x514`

Reply:
OK,I will fix that.